var/spack/repos/builtin/packages/lua-lpeg/package.py

```diff
@@ -16,8 +16,7 @@ class LuaLpeg(Package):
     version('1.0.2-1', sha256='e0d0d687897f06588558168eeb1902ac41a11edd1b58f1aa61b99d0ea0abbfbc', expand=False)
     version('0.12-1', sha256='3962e8d695d0f9095c9453f2a42f9f1a89fb94db9b0c3bf22934c1e8a3b0ef5a', expand=False)
 
-    extends("lua")
-    depends_on("lua@:5.1.9", when="@:0.12.1")
+    depends_on("lua@:5.1.9", when="@:0.12.1^lua")
 
     def install(self, spec, prefix):
```

Comment:
Does the default install method not work?

Reply:
Thanks for the reminder.  This version of the package would not have worked, no, because lpeg was using the actual rock archive from the luarocks distro rather than a source package.  Basically a luarocks wheel.  I'm reworking it to use the source instead then the default works just fine.

Reply:
Spack now supports the rock archives directly, and this is reworked to use it.